### DISCUSSION TITLE
Unable to read config file on restart

### DISCUSSION THREAD

**Discussion by hchawale** [dmunozv04/iSponsorBlockTV#100] [Q&A]
Hi! I need to relink the TV and rebuild the config file every time I restart my computer. 

I'm using Docker (Windows) and get a config not found error. Exiting in 10 seconds error.

Any chance you can help in persisting the config file? I swear this can be done thru the Directory-to-app variable, but would appreciate some hand holding

**Comment by dmunozv04** [comment]
I have not used docker desktop much myself but can't you just use the instructions for running on docker on linux from windows' cmd/terminal? 
You'd run the `docker run` command with the `--mount` argument pointing to a folder, and that would persist container restarts/updates
